cognite/client/_api/relationships.py

```diff
@@ -32,36 +33,6 @@ def __init__(self, config: ClientConfig, api_version: str | None, cognite_client
         super().__init__(config, api_version, cognite_client)
         self._LIST_SUBQUERY_LIMIT = 1000
 
-    def _create_filter(
```

Comment:
Removed because it is stupid? Just do a big call to do another big call

Reply:
Yeah, was completely unneccessary 😅